shared/rocroller/lib/source/KernelGraph/Transformations/AliasDataFlowTags.cpp

```diff
@@ -265,28 +265,33 @@ namespace rocRoller
                         auto        aIdx = graphIndices[a];
                         auto        bIdx = graphIndices[b];
 
-                        auto order = kgraph.control.compareNodes(
-                            rocRoller::UpdateCache, aRec.control, bRec.control);
-
-                        switch(order)
+                        if(aRec.control != bRec.control)
```

Comment:
Would "no nesting" work?

Also, a comment would be helpful.

```suggestion
                        // This can happen with Exchange nodes because EXPLAIN
                        if(aRec.control == bRec.control)
                            continue;
```

Then you don't have to indent the rest.